skate-plugin/src/test/kotlin/com/slack/sgp/intellij/modeltranslator/TranslatorHelperTest.kt

```diff
@@ -130,4 +156,6 @@ class TranslatorHelperTest : LightJavaCodeInsightFixtureTestCase() {
   private fun generateBody(): KtBlockExpression? {
     return TranslatorHelper.generateBody(TranslatorHelper.extractBundle(getNamedFunction())!!)
   }
+
+  private fun skatePluginSettings() = project.service<SkatePluginSettings>()
```

Comment:
We have an extension for this here: https://github.com/slackhq/slack-gradle-plugin/blob/main/skate-plugin/src/main/kotlin/com/slack/sgp/intellij/util/ProjectUtils.kt#L22